namespaces/Representor.md

```diff
@@ -0,0 +1,107 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Object Element)
```

Comment:
I think `Element` instead of `Object Element` is sufficient here, because you are overriding the rest anyway. Or am I missing something?